app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss

```diff
@@ -309,7 +309,7 @@
 
       &.is-private {
         background: lighten($warning-color, 32%);
-        border: 1px solid $color-border;
+        border: 1px solid $color-warning;
```

Comment:
Can we try a little more lightened version like the background ?

Reply:
Tuned a little bit, `lighten(#ffc532, 15%) = #ffdb7e`

What are your thoughts?

![Screen Shot 2020-07-01 at 7 13 06 PM](https://user-images.githubusercontent.com/7905931/86272629-2ca72f80-bbcf-11ea-816f-09549dfbe2f5.png)

Reply:
Looks good, Make sure you are using `$warning-color` for consistency.

Reply:
Alright 👍🏼 ..
> Make sure you are using `$warning-color` for consistency.

That's correct, it is `$warning-color` used there (`lighten(#ffc532, 15%)`), I don't know why I wrote it `$color-warning` tho, apologies.

Will submit changes in a few moments.